traffic_ops/traffic_ops_golang/cachesstats/cachesstats.go

```diff
@@ -146,11 +146,19 @@ func addStats(cacheData []CacheData, stats tc.Stats) []CacheData {
 		}
 		bandwidth, ok := stat.Stats[tc.StatNameBandwidth]
 		if ok && len(bandwidth) > 0 {
-			cache.KBPS = bandwidth[0].Val.(uint64)
+			cache.KBPS, ok = bandwidth[0].Val.(uint64)
```

Comment:
So when I was looking at this, it seemed like TM was serving this data as strings. If that's the case, we would be hitting this warning case every time. Maybe we need to actually convert the string to a uint64 using `strconv` instead of casting it?

Reply:
On the other hand, was that a breaking API change? Maybe TM should serve integers instead of strings? I'm not sure, though, if/why that changed

Reply:
I don't know, from the small amount of digging I did, it seemed like the TM API has always served strings.

Reply:
So TM serving strings for stats is expected?

Reply:
I believe so. We should `strconv` it